lib/class-wp-theme-json-gutenberg.php

```diff
@@ -2093,6 +2097,15 @@ protected static function compute_style_properties( $styles, $settings = array()
 				$value = gutenberg_get_typography_font_size_value( array( 'size' => $value ) );
 			}
 
+			if ( 'aspect-ratio' === $css_property ) {
+				// For aspect ratio to work, other dimensions rules must be unset.
+				// This ensures that a fixed height does not override the aspect ratio.
+				$declarations[] = array(
+					'name'  => 'min-height',
+					'value' => 'unset',
+				);
+			}
+
```

Comment:
Yes — we need the `min-height: unset` in global styles output (which is this part of the code) so that we unset the Cover block (or any other block's) default min-height value that might be in use.